### PR TITLE
[frio] Redo right offcanvas menu

### PR DESCRIPTION
**Pull request by Extarys**
This is a test. It only replaces the right menu on mobile for now. If everything is okay I'll later do the same to the left offcanvas menu.

I also changed the color of the search icon in mobile view.

This is a start for #9837 

Result:
image

Side note: I'll soon make a PR to include prettier/eslint config to the frio folder for consistency in style. VS Code is dumb.

### CODE REVIEW

File: view/theme/frio/js/theme.js [hunk 1]
```diff
@@ -359,22 +366,45 @@ $(document).ready(function(){
...
-		.on("shown.bs.offcanvas", function() {
+		.on("shown.bs.offcanvas", function () {
 			$body.addClass("aside-out");
 		})
-		.on("hidden.bs.offcanvas", function() {
+		.on("hidden.bs.offcanvas", function () {
 			$body.removeClass("aside-out");
 		});
 
+	// Right offcanvas elements
```
Comment (Extarys) [inline view/theme/frio/js/theme.js:376]: Code change starts here. Nothing else was changed in this file.

**Comment by MrPetovan** [comment]
So, we had issues with fixing the formatting of files that are changed in a PR before, and the solution if you can't prevent your IDE from reformatting the files you're touching is to commit the reformatted files *without your changes* first, and then submitting your changes in additional commits. This allows a commit-by-commit review of the PR without having to guess where the real changes are and where the formatting changes are.

**Review by MrPetovan — CHANGES_REQUESTED** [review]
So, we had issues with fixing the formatting of files that are changed in a PR before, and the solution if you can't prevent your IDE from reformatting the files you're touching is to commit the reformatted files without your changes first, and then submitting your changes in additional commits. This allows a commit-by-commit review of the PR without having to guess where the real changes are and where the formatting changes are.

**Comment by Extarys** [comment]
Wise.

Can you tell me how I can "replay" these changes on top of the current develop version (HEAD?) once #9850 is merged?

**Comment by MrPetovan** [comment]
Ha, you won't like it, because it will involve `git rebase` and probably fixing conflicts since I've been asking you some changes over at #9850.

Depending on the extensiveness of the changes in `nav.tpl`, you may be better off discarding all the changes in the file and starting over once this branch is rebased on top of the updated `develop` after the formatting PR is merged.

Basically, it would go like this once the PR is merged:
```
# update your friendica remote with the updated develop branch
> git fetch --all

# update your local develop branch to the friendica state and push it to your fork
> git checkout develop && git rebase friendica/develop && git push

# Switch to your working branch for this current PR
> git checkout offcanvas

# Replay your branch on top of the updated develop
> git rebase develop

# CONFLICTS! Good luck.

# Once you're done, you need to force-push your branch since you're rewriting history.
# It's fine in this case because it's your branch on your fork. It's more problematic with a shared repo.
> git push --force
```

**Comment by Extarys** [comment]
@MrPetovan , the Git Master.

There are some ashes here and there of the old offcanvas that I'll clean up when I change the left offcanvas. Either today or tomorrow, I'll wait a little once this is merged in case mobile users notice breaking bugs. (Shouldn't happen, but better be safe than sorry)

File: view/theme/frio/templates/nav.tpl [hunk 2]
```diff
@@ -28,15 +28,15 @@
 				<div class="topbar-nav" role="navigation">
 
 					{{* Buttons for the mobile view *}}
-					<button type="button" class="navbar-toggle collapsed pull-right" data-toggle="offcanvas"
-						data-target="#myNavmenu" aria-controls="myNavmenu" aria-haspopup="true">
+					<button type="button" class="navbar-toggle offcanvas-right-toggle pull-right" aria-controls="myNavmenu"
```
Comment (MrPetovan) [inline view/theme/frio/templates/nav.tpl]: The `aria-controls` attribute must be updated.
```suggestion
					<button type="button" class="navbar-toggle offcanvas-right-toggle pull-right" aria-controls="offcanvasUsermenu"
```

**Review by MrPetovan — CHANGES_REQUESTED** [review]

**Review by MrPetovan — APPROVED** [review]

**Pull request merged**